packages/sdk/src/index.ts

```diff
@@ -93,42 +94,48 @@ export class Gomu {
   }
 
   async makeOrder(params: MakeOrderParams): Promise<Order[]> {
```

Comment:
`async makeOrder({ marketplaces, ...params}: MakeOrderParams): Promise<Order[]> {`
Slightly more correct because `marketplace.makeOrder` doesn't have marketplaces in its params. Also don't repeat the `params.marketplaces`